robolectric/src/test/java/org/robolectric/shadows/ShadowLauncherAppsTest.java

```diff
@@ -27,6 +31,7 @@
 import org.junit.runner.RunWith;
 import org.robolectric.annotation.Config;
 import org.robolectric.shadow.api.Shadow;
+import org.robolectric.util.ReflectionHelpers;
 import org.robolectric.util.reflector.Accessor;
 import org.robolectric.util.reflector.ForType;
 
```

Comment:
Can the `@Config(minSdk = O_MR1)` in this test file changed to `L`? You may have to adjust some individual tests to be `@Config(minSdk = O_MR1)`.

Reply:
The minimum sdk can used is N for test, otherwise setup will throw exception for non exist service.